src/Commands/core/UpdateDBCommands.php

```diff
@@ -238,7 +238,10 @@ public static function updateDoOne($module, $number, array $dependency_map, Drus
             }
         } else {
             $ret['#abort'] = ['success' => false];
-            Drush::logger()->warning(dt('Update function @function not found', ['@function' => $function]));
+            Drush::logger()->warning(dt('Update function @function not found in file @filename', [
+                '@function' => $function,
+                '@filename' => "$module.install.php",
```

Comment:
Copy/paste snafu - extra trailing `.php`

Reply:
Fixed, thanks for reviewing!